sources/packages/backend/libs/sims-db/src/pagination-utils.ts

```diff
@@ -0,0 +1,28 @@
+import { FieldSortOrder } from "@sims/utilities";
+import { OrderByCondition } from "typeorm";
+
+ /**
+   * Transformation to convert the data table column name to database column name.
+   * Any changes to the data object (e.g data table) in presentation layer must be adjusted here.
+   * @param sortField database fields to be sorted.
+   * @param sortOrder sort order of fields (Ascending or Descending order).
+   * @returns OrderByCondition
+   */
+ export function transformToEntitySortField(
```

Comment:
Is this method supposed to be generic?
When we need to use column references like `application.applicationNumber` I would assume that it would be dedicated to certain queries and it should be named accordingly.